config/locales/refunds.yml

```diff
@@ -160,7 +160,7 @@ en:
         For audit
       applicant_personal_information_usage_header: "Using your personal information:"
       applicant_personal_information_usage_sub_header: "Personal information which you supply to us may be used in a number of ways, for example:"
-      name_changed_cannot_continue: "If your name has changed since you submitted your original Employment Tribunal claim you will need to complete a paper form which can be downloaded here. You will be asked to provide evidence of your name change."
+      name_changed_cannot_continue_html: 'If your name has changed since you submitted your original Employment Tribunal claim you will need to complete a paper form which can be downloaded <a target="_blank" href="https://hmctsformfinder.justice.gov.uk/HMCTS/GetForm.do?original_id=2033">here</a>. You will be asked to provide evidence of your name change.'
```

Comment:
Can you use some kind of helper method instead of having an HTML in translation file?

Reply:
Html is in the yml file in a lot of places so we should put in a ticket for that one

Reply:
Sometimes html in a translation file is good - what about if the link needed to change for a welsh version for example ?

Reply:
I've seen both approaches in our yaml files 
https://github.com/ministryofjustice/atet/blob/master/config/locales/en.yml#L460
https://github.com/ministryofjustice/atet/blob/master/config/locales/en.yml#L433
but I think I like a bit more the helper method way.